File: Day-33/day33.cpp
#include<iostream>
#include <climits>
using namespace std;

int main(){
    int MinDif = INT_MAX;
    int diff = 0 ;
    int arr[5] = {1, 5, 3, 19, 18};

    for(int i=0;i<5;i++){
        for(int j=i+1;j<5;j++){
            if(arr[i]!= arr[j]){
                diff = abs(arr[i] - arr[j]);
                if(diff < MinDif){
                    MinDif = diff;
                }
            }
        }
    }
    cout<<"Minimum Difference is: "<<MinDif<<endl;
    return 0;
}

File: Day-33/day33.py
nums = [4,7,2,9,8,1,5,6,3]

min_diff = float('inf')

for i in range(len(nums)):
    for j in range(i + 1, len(nums)):
        diff = abs(nums[i] - nums[j])
        if (diff <min_diff):
            min_diff = diff
print("The minimum difference is:", min_diff)